ssas/systems.go

```diff
@@ -220,42 +219,27 @@ func (system *System) GetSecret(ctx context.Context) (Secret, error) {
 }
 
 // SaveTokenTime puts the current time in systems.last_token_at
-func (system *System) SaveTokenTime(ctx context.Context) {
-	event := Event{Op: "UpdateLastTokenAt", TrackingID: system.GroupID, ClientID: system.ClientID}
-	OperationCalled(event)
-
-	err := Connection.WithContext(ctx).Model(&system).UpdateColumn("last_token_at", time.Now()).Error
+func (system *System) SaveTokenTime(ctx context.Context) (err error) {
+	err = Connection.WithContext(ctx).Model(&system).UpdateColumn("last_token_at", time.Now()).Error
 	if err != nil {
-		event.Help = err.Error()
-		OperationFailed(event)
+		return err
 	}
-
-	OperationSucceeded(event)
+	return nil
 }
 
 /*
 RevokeSecret revokes a system's secret
 */
-func (system *System) RevokeSecret(ctx context.Context, trackingID string) error {
-	revokeCredentialsEvent := Event{Op: "RevokeCredentials", TrackingID: trackingID, ClientID: system.ClientID}
-	OperationStarted(revokeCredentialsEvent)
-
-	xdata, err := XDataFor(ctx, *system)
+func (system *System) RevokeSecret(ctx context.Context, systemID string) error {
+	_, err := XDataFor(ctx, *system)
 	if err != nil {
-		revokeCredentialsEvent.Help = fmt.Sprintf("could not get group XData for clientID %s: %s", system.ClientID, err.Error())
-		OperationFailed(revokeCredentialsEvent)
-		return fmt.Errorf("unable to find group for clientID %s: %s", system.ClientID, err.Error())
+		return fmt.Errorf("could not get group XData for clientID %s: %s", system.ClientID, err.Error())
 	}
 
 	err = system.deactivateSecrets(ctx)
 	if err != nil {
-		revokeCredentialsEvent.Help = "unable to revoke credentials for clientID " + system.ClientID
-		OperationFailed(revokeCredentialsEvent)
-		return fmt.Errorf("unable to revoke credentials for clientID %s: %s", system.ClientID, err.Error())
+		return fmt.Errorf("unable to revoke credentials for clientID " + system.ClientID)
 	}
-
```

Comment:
Do we lose anything with the removal of this line? Only one where I don't see a log transferred along in the new version

Reply:
I see it, upstream.